src/tests/conformance/handshake.rs

```diff
@@ -23,18 +23,23 @@ async fn c001_handshake_when_node_receives_connection() {
         .await
         .expect("Unable to build a synthetic node");
 
+    let net_addr = node.net_addr().expect("Network address not found");
```

Comment:
I believe the convention is to have these be lower case?

Reply:
I usually don't pay much attention to capitalization in panic conditions, but yeah, consistency won't hurt :ok_hand:

Reply:
> I believe the [convention](https://doc.rust-lang.org/std/error/index.html#common-message-styles) is to have these be lower case?

Yes! I will append one more commit here in this PR to fix this across the repository. Thanks 🙂

Reply:
Aaand done